grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -602,6 +602,11 @@ class OrderController {
         }
         else {
             orderItem.properties = params
+            Shipment pendingShipment = order.pendingShipment
+            if (pendingShipment) {
+                ShipmentItem itemToUpdate = pendingShipment.shipmentItems.find { it.orderItemId == orderItem.id }
```

Comment:
This should probably apply to all shipment items that are associated with the order item, so change find to findAll

Reply:
@jmiranda You're right. I've updated PR.